### ISSUE TITLE
How to disable select mode for the language servers?

### ISSUE THREAD

**Issue by toluschr** [neoclide/coc.nvim#1718]
First of all, providing a solution to my problem with select mode is not necessary, if someone tells me how to disable select mode for the `coc-languageserver` completely / use visual mode instead.
## versions

vim version: VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Mar 14 2020 07:37:32)
node version: v13.2.0
coc.nvim version: 0.0.77-817efd5108
term: dumb
platform: linux

## Output channel: languageserver.clangd

I[13:22:10.817] clangd version 9.0.1 
I[13:22:10.817] Working directory: /home/tom/Code/test
I[13:22:10.817] argv[0]: clangd
I[13:22:10.817] argv[1]: --background-index
I[13:22:10.817] Starting LSP over stdin/stdout
I[13:22:10.817] <-- initialize(0)
I[13:22:10.817] --> reply:initialize(0) 0 ms
I[13:22:10.821] <-- initialized
I[13:22:10.821] unhandled notification initialized
I[13:22:10.822] <-- textDocument/didOpen
I[13:22:10.823] Failed to find compilation database for /home/tom/Code/test/main.c
I[13:22:10.823] Updating file /home/tom/Code/test/main.c with command clangd fallback
[/home/tom/Code/test]
/usr/bin/clang /home/tom/Code/test/main.c -fsyntax-only -resource-dir=/usr/lib/clang/9.0.1
I[13:22:10.842] --> textDocument/publishDiagnostics

## Messages
Messages maintainer: Bram Moolenaar <Bram@vim.org>

## Describe the bug
When using any language server and selecting the first entry in the recommendations, the first argument gets selected. Typing something replaces the selected text. A single press of the escape-key should get me out of the mode.
I have to press escape twice, otherwise pressing (in this case) the arrow key is buggy (See screenshot).

## Reproduce the bug
My vimrc
```vim
syntax on
filetype plugin indent on
set shiftwidth=0
set tabstop=1
set number

" Disable netrw
let loaded_netrwPlugin=1
set autoindent

call plug#begin('~/.vim/plugged')
Plug 'neoclide/coc.nvim', {'branch': 'release'}
call plug#end()

" Enable omnicompletion
set omnifunc=syntaxcomplete#Complete

nmap <space>e :CocCommand explorer<CR>

" Go to definition
nmap <silent> gd :call CocAction('jumpDefinition', 'split')<CR>

" Alternative to colorcolumn
match Error /\%>90c/

" Hybrid Line Numbers
:set number relativenumber
:set nu rnu
```
Mini.vim doesn't load clangd

## Screenshots (optional)
![Peek 2020-04-04 12-56](https://user-images.githubusercontent.com/44272603/78438647-8bfcae80-7677-11ea-93d1-a9495f688c3c.gif)

**Comment by chemzqm** [comment]
No, because you have to rename placeholder most of the time, just type characters or jump to next placeholder

**Comment by toluschr** [comment]
I'm not sure, you understood my issue, so let me rephrase this, as this problem still exists:
I type characters (As you told me to) and press `<Esc>`.
Now I am out of insert mode and everything works fine.
However, as soon as I use the arrow keys instead of hjkl, it prints a character.

**Comment by toluschr** [comment]
```
nnoremap <silent> <ESC>OA <UP>
nnoremap <silent> <ESC>OB <DOWN>
nnoremap <silent> <ESC>OC <RIGHT>
nnoremap <silent> <ESC>OD <LEFT>
```
Fixed it
